vision/mnist/vae.jl

```diff
@@ -47,19 +47,20 @@ modelsample() = rand.(Bernoulli.(f(z.(zeros(Dz), zeros(Dz)))))
 ################################# Learn Parameters ##############################
 
 evalcb = throttle(() -> @show(-L̄(X[:, rand(1:N, M)])), 30)
-opt = ADAM(params(A, μ, logσ, f))
+opt = ADAM()
+ps = params(A, μ, logσ, f)
 @progress for i = 1:20
   @info "Epoch $i"
-  Flux.train!(loss, zip(data), opt, cb=evalcb)
+  Flux.train!(loss, ps, zip(data), opt, cb=evalcb)
 end
 
 
 ################################# Sample Output ##############################
 
-using Images
+#using Images
```

Comment:
Why is this getting commented out?

Reply:
This was an artefact of debugging the notebooks generation. Thanks for catching it